python/ray/air/util/data_batch_conversion.py

```diff
@@ -109,6 +109,58 @@ def convert_pandas_to_batch_type(
         )
 
 
+def _convert_batch_type_to_numpy(
```

Comment:
I think we should just move this entire file into `air._internal` and remove the DeveloperAPI from the convert pandas functions. 

The convert pandas don't need to be Developer API anymore now that we have `DLPredictor`.

Reply:
discussed offline -- let's do it in separate PR